README.rst

```diff
@@ -375,6 +375,32 @@ Usage exmaple:
     >>> user.organizations_dict.keys()
     ... dict_keys(['20135c30-d486-4d68-993f-322b8acb51c4'])
 
+Organization membership permission_classes
+------------------------------------------
+
+The custom rest_framework permission classes ``IsOrganizationMember``,
+``IsOrganizationManager`` and ``IsOrganizationOwner`` can be used in the
+API to ensure that the request user is in the same organization as requested
+object and is organization member, manager or owner respectively.
+
+Usage exmaple:
+
+.. code-block:: python
```

Comment:
Please check how it looks:

https://github.com/atb00ker/openwisp-users/tree/issues/150-org-member-validator#organization-membership-permission_classes